mteb/tasks/Retrieval/multilingual/XPQARetrieval.py

```diff
@@ -74,33 +69,107 @@ class XPQARetrieval(MultilingualTask, AbsTaskRetrieval):
         },
         type="Retrieval",
         category="s2p",
-        eval_splits=[_EVAL_SPLIT],
-        eval_langs=_LANGS,
+        eval_splits=["test"],
+        eval_langs=_EVAL_LANGS,
         main_score="ndcg_at_10",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
-        n_samples=None,
-        avg_character_length=None,
+        date=("2022-01-01", "2023-07-31"),  # best guess
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Question answering"],
+        license="CDLA-Sharing-1.0",
+        socioeconomic_status="mixed",
+        annotations_creators="human-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""@inproceedings{shen2023xpqa,
+        title={xPQA: Cross-Lingual Product Question Answering in 12 Languages},
+        author={Shen, Xiaoyu and Asai, Akari and Byrne, Bill and De Gispert, Adria},
+        booktitle={Proceedings of the 61st Annual Meeting of the Association for Computational Linguistics (Volume 5: Industry Track)},
+        pages={103--115},
+        year={2023}
+        }""",
+        n_samples={"test": 19801},
+        avg_character_length={"test": 104.68},  # answer
```

Comment:
Now that this metadata is filled, we can remove this from `_HISTORIC_DATASETS` of `tests/test_TaskMetadata.py`

Reply:
will give it a try